Add_Credential_Jenkins.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# This script reads key-value pairs from a file and creates a Jenkins credential
+
+# Replace the following variables with your Jenkins configuration
+JENKINS_URL="http://localhost:8080"
+CREDENTIAL_ID="your_credential_id"
+
+# Check if the API token and username were passed as command-line arguments
+if [ $# -ne 2 ]; then
+  echo "Usage: $0 <USER_NAME> <API_TOKEN>"
+  exit 1
+fi
+
+# Assign the command-line arguments to variables
+USER_NAME="$1"
+API_TOKEN="$2"
+
+# Check if the file containing key-value pairs exists and is readable
+if [ ! -r "secrets.txt" ]; then
+  echo "File 'secrets.txt' not found or not readable."
+  exit 1
+fi
+
+# Read each line from the input file and create a credential for each key-value pair
+while read line; do
+  key=$(echo $line | cut -d'=' -f1)
+  value=$(echo $line | cut -d'=' -f2)  
+  
+# Check if a credential with the same ID already exists
+  existing_credential=$(curl -s -o /dev/null -w "%{http_code}" -X GET -u "${USER_NAME}:${API_TOKEN}" "${JENKINS_URL}/credentials/store/system/domain/_/credential/${CREDENTIAL_ID}_${key}/api/json")
+  if [ ${existing_credential} -eq 200 ]; then
+    echo "Credential '${key}' already exists. Skipping."
```

Comment:
Don't skip, instead update the existing with whatever we have in on file..we will mark it as a source of truth.